test/unit/ut_crypto.c

```diff
@@ -280,6 +281,7 @@ UTEST(CRYPTO_C, EXT_PROC_PDU)
     status = Crypto_Process_Extended_Procedure_Pdu(tc_frame, ingest);
     ASSERT_EQ(status, CRYPTO_LIB_SUCCESS);
 }
+#endif //CRYPTO_EPROC
```

Comment:
Can we expand this with an `#elseif` to test that extended procedures is not accepted when commanded if disabled?